### PR TITLE
Change details panel to element inspector

### PR DESCRIPTION
**Pull request by nidasid**
- [x] @joeferner
- [x] @diegogrz
- [x] @mwizeman @sfeng88
- [x] @joeybrk372 @rygim @jharwig @EvanOxfeld

Testing Instructions: Verify that details panel now says element inspector

Points of Regression: Anywhere that would reference the details panel

CHANGELOG
Added: 
Changed: 
Documentation: Rename `details panel` to `element inspector`
Deprecated:
Removed:
Fixed:
Security:

### CODE REVIEW

**Comment by jharwig** [comment]
Can you undo all changes to files under `docs/extension-points/front-end`? Those files are all completely changed in another PR

**Comment by nidasid** [comment]
@jharwig done.

File: web/plugins/structured-ingest/core/src/main/resources/user-guide/csv/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Visallo can import CSVs and extract entities from the data. There is a custom wi
 
 <img src='./drag-csv-into-visallo.gif' />
 
-<span>2. Once the file shows a button that says "Map Structured Data" in the [detail panel](../detail-pane.md), click that button and the csv/excel import wizard will appear.</span>
+<span>2. Once the file shows a button that says "Map Structured Data" in the [element inspector](../detail-pane.md), click that button and the csv/excel import wizard will appear.</span>
```
Comment (EvanOxfeld) [inline web/plugins/structured-ingest/core/src/main/resources/user-guide/csv/index.md]: Let's rename `detail-pane.md` to `element-inspector.md`

**Comment by nidasid** [comment]
@EvanOxfeld done

**Pull request merged**